swervelib/Mk4iSwerveModuleHelper.java

```diff
@@ -1,23 +1,25 @@
 package frc.team670.mustanglib.swervelib;
 
 import frc.team670.mustanglib.swervelib.ctre.*;
+import frc.team670.mustanglib.swervelib.redux.*;
```

Comment:
I'm going to stop reading here. I don't understand why all these unrelated changes are in here. Let's figure that out and then finish this